libraries/SITL/SIM_Vicon.h

```diff
@@ -13,7 +13,22 @@
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 /*
-  simple particle sensor simulation
+  simple Vicon simulation
+
+  NoopLoop testing:
+
+./Tools/autotest/sim_vehicle.py --gdb --debug -v ArduCopter -A --uartF=sim:vicon --speedup=1
+
+param set GPS_TYPE 0
+param set EK2_GPS_TYPE 3
```

Comment:
I'm not sure it's a good idea to have these param values here.  It's probably better instead to have them on the wiki "Using SITL" page.  In any case I think we recommend using EKF3 these days.